api/stripe.go

```diff
@@ -229,16 +230,20 @@ func (api *API) processStripeSub(ctx context.Context, s *stripe.Subscription) er
 		if subsc == nil || (mostRecentSub != nil && subsc.ID == mostRecentSub.ID) {
 			continue
 		}
+		if subsc.ID == "" {
+			api.staticLogger.Debugf("Empty subscription ID! User ID '%s', Stripe ID '%s', subscription object '%+v'", u.ID.Hex(), u.StripeID, subs)
```

Comment:
`nit` I would bump the severity here to a warning or an error even, this is not supposed to happen and if it does it warrants an investigation I would say

Reply:
At the very least I wouldn't make it a Debug print. It's definitely an error.

Reply:
It's not an error and we can't do anything about it. It's all about data coming from Stripe. It's completely non-actionable on our part.

I bumped it to Warning - we'll see it in the logs.